test/FastTests/Issues/RavenDB_9519.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Commands;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Http;
+using Raven.Server.Utils;
+using Sparrow.Json;
+using Xunit;
+
+namespace FastTests.Issues
+{
+    public class RavenDB_9519 : RavenTestBase
+    {
+        [Fact]
+        public async Task NestedObjectShouldBeExportedAndImportedProperly()
+        {
+            string tmpFile = null;
+            try
+            {
+                using (var store = GetDocumentStore())
+                {
+                    using (var session = store.OpenSession())
+                    {
+                        session.Store(_testCompany, "companies/1");
+                        session.SaveChanges();
+                    }
+
+                    var client = new HttpClient();
+                    var stream = await client.GetStreamAsync($"{store.Urls[0]}/databases/{store.Database}/streams/queries?query=From%20companies");
+                    tmpFile = Path.GetTempFileName();
+                    using (var file = File.Create(tmpFile))
+                    {
+                        stream.CopyTo(file);
+                        await file.FlushAsync();
+                    }
+
+                    using (var commands = store.Commands())
+                    {
+                        var getOperationIdCommand = new GetNextOperationIdCommand();
+                        await commands.RequestExecutor.ExecuteAsync(getOperationIdCommand, commands.Context);
+                        var operationId = getOperationIdCommand.Result;
+
+                        using (var fileStream = File.OpenRead(tmpFile))
+                        {
+                            var csvImportCommand = new CsvImportCommand(fileStream, null, operationId);
+
+                            await commands.ExecuteAsync(csvImportCommand);
+
+                            var operation = new Operation(commands.RequestExecutor, () => store.Changes(), store.Conventions, operationId);
+
+                            await operation.WaitForCompletionAsync();
+                        }
+                    }
+
+                    using (var session = store.OpenSession())
+                    {
+                        var res = session.Query<Company>().ToList();
+                        Assert.Equal(2, res.Count);
+                        Assert.True(res[0].Equals(res[1]));
```

Comment:
I couldn't make this assertion to pass, can you check that?

Reply:
The actual issue here is in `Address.Line2`.
The problem is that we have a null value vs. an empty string, and the question here is how do we handle the difference between them.